TrashMob/client-app/package.json

```diff
@@ -40,13 +42,17 @@
     "react-simple-captcha": "^9.0.2",
     "reactstrap": "^8.9.0",
     "rimraf": "^3.0.2",
+    "sass": "^1.79.3",
+    "vite": "^5.4.7",
+    "vite-plugin-svgr": "^4.2.0",
+    "vite-tsconfig-paths": "^5.0.1",
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "vite",
+    "build": "vite build",
+    "test": "vite test",
+    "eject": "vite eject"
```

Comment:
There's no `vite eject`, can remove this command.

Reply:
Removed. Thanks.